Test/tester.mgf

```diff
@@ -0,0 +1,1174 @@
+BEGIN IONS
```

Comment:
rather than including this .mgf in mzlib, can you convert the existing tester.mzml to mgf in your unit test, then assert to check the properties? then you can see if reading mgf works but also preserves fidelity between mzml and mgf while making minimal changes to existing unit tests and not having to add any additional mgf files or remove the old tester.mzml

Reply:
I'm a little confused by what you're asking. The test is checking the ability to load mgfs, not to convert an mzml into an mgf? 
If we did write code to convert mzml to mgf, couldn't we accidentally write it incorrectly and thus read it incorrectly? It would just be testing our ability to make up a format and then read that format, not necessarily an mgf.
I haven't changed any existing unit tests; I removed one unnecessary assert to trim an mzml.
I didn't remove the old tester.mzml, I just trimmed it in the same way I trimmed the mgf.

Reply:
ah, I just realized you're not adding writing MGF. ignore my previous comment